include/swift/AST/DiagnosticsCommon.def

```diff
@@ -54,6 +54,10 @@ NOTE(previous_decldef,none,
 NOTE(brace_stmt_suggest_do,none,
      "did you mean to use a 'do' statement?", ())
 
+NOTE(defer_stmt_at_block_end,none,
```

Comment:
This should be a warning.  Notes are always attached to warnings or errors.

Reply:
This is fixed in the next commit :)